Команда 4

kossakovskaia opened the pull request.

lisiza committed:
+ html structure and fonts

lisiza committed:
+ normalize.css

imynel committed:
Merge pull request #7 from lisiza/lm_layout

+ html structure and fonts

imynel committed:
add pictures

lisiza committed:
+readme

kossakovskaia committed:
Merge pull request #8 from imynel/master

add pictures

kossakovskaia committed:
Merge pull request #9 from lisiza/lm_readme

+readme

kossakovskaia committed:
Add gitignore, editorconfig, favicon and script

lisiza committed:
Merge pull request #11 from kossakovskaia/ik_edit-base-setup

Add gitignore, editorconfig, favicon and script

kossakovskaia committed:
Delete ds_store files

kossakovskaia committed:
Delete redundant files

kossakovskaia committed:
Add .nojekyll for bem

kossakovskaia committed:
Add common margin and paddings

kossakovskaia committed:
Add block class for all sections

imynel committed:
Merge pull request #27 from kossakovskaia/ik_edit-common-paddings

Edit common paddings

imynel committed:
init section-about

Sirius-CMa committed:
Добавлен стиль курсора

lisiza committed:
+ Agency block

lisiza committed:
+ agency block ID

imynel committed:
add section-projects

imynel committed:
add section in index.css

kossakovskaia committed:
Add digits section for desktop only

kossakovskaia committed:
Add tablet and mobile views

kossakovskaia committed:
Fix intermediate state

kossakovskaia committed:
Fix fonts

lisiza committed:
fixed review points

Sirius-CMa committed:
Изменено: курсор, прыжки border, сокращено написание стилей

lisiza committed:
started a stories feature

imynel committed:
Merge pull request #29 from kossakovskaia/an-button-test

An button test

kossakovskaia committed:
Resolve conflicts

kossakovskaia committed:
Add is and aria-label

lisiza committed:
working on feature

lisiza committed:
get lost

lisiza committed:
seem to resolve conflicts

lisiza committed:
Merge pull request #30 from kossakovskaia/lm_agency

Блок "Лучшие проекты агентства"

kossakovskaia committed:
Resolve conflicts

kossakovskaia committed:
Merge pull request #32 from kossakovskaia/ik_main-numbers

Секция "Everland в цифрах"

Commit:
add block for-whom

kossakovskaia committed:
Fix incorrect html after merging branches

imynel committed:
resolve conflict

imynel committed:
merge develop branch

imynel committed:
improved layout

imynel committed:
add new btn

imynel committed:
add button and update layout

imynel committed:
Merge branch 'develop' of github.com:kossakovskaia/everland into NS_section-projets

imynel committed:
add btn in section

kossakovskaia committed:
Fix btn

kossakovskaia committed:
Fix text for btn

kossakovskaia committed:
Add donate block

kossakovskaia committed:
Add daonator list desktop

kossakovskaia committed:
Fix gradient

imynel committed:
Merge branch 'develop' of github.com:kossakovskaia/everland into NS_section-projets

Sirius-CMa committed:
Гит упёрся, просит коммит

Commit:
пересобрал карточки

Sirius-CMa committed:
Merge branch 'develop' of https://github.com/kossakovskaia/everland into develop

Sirius-CMa committed:
Пробный коммит

Sirius-CMa committed:
Дополнены Header, Menu, accordeons

Commit:
внес правки

Commit:
add block work-directions

lisiza committed:
+ media

lisiza committed:
Merge branch 'develop' of github.com:kossakovskaia/everland into lm_stories

lisiza committed:
Merge pull request #38 from kossakovskaia/lm_stories

+ Our stories block

kossakovskaia committed:
Merge develop

Commit:
fix cards

Commit:
fix before merge

sashaslab committed:
Merge pull request #33 from kossakovskaia/as_for-whom-section

add block for-whom

kossakovskaia committed:
Edit typography

kossakovskaia committed:
Merge develop

Commit:
merge develop branch

Sirius-CMa committed:
Merge commit 'bdb1cd057ce08dd47d46eb5385705dd5dfa0ffd4' into develop

Commit:
fix card

imynel committed:
fix conflict

Sirius-CMa committed:
Header, menu, accordeons

Sirius-CMa committed:
Merge pull request #40 from kossakovskaia/an-header-menu-3

An header menu 3

Commit:
fix card

kossakovskaia committed:
Bem form organization

Commit:
fixing bugs

Commit:
fixing bugs

sashaslab committed:
Merge pull request #41 from kossakovskaia/as_work-directions

As work directions

Commit:
fixin bugs card

kossakovskaia committed:
Merge develop

sashaslab committed:
Merge pull request #39 from kossakovskaia/as_for-whom-section

fix card

imynel committed:
add adaptive

Commit:
fix title section

kossakovskaia committed:
Merge develop

Commit:
Merge branch 'develop' of github.com:kossakovskaia/everland into as_work-directions

Commit:
fix

sashaslab committed:
Merge pull request #42 from kossakovskaia/as_work-directions

As work directions

kossakovskaia committed:
Fix id's

kossakovskaia committed:
Add heading styles

imynel committed:
fix conflict

imynel committed:
fix conflict

imynel committed:
Merge branch 'develop' of github.com:kossakovskaia/everland into NS_section-about

imynel committed:
Merge pull request #31 from kossakovskaia/NS_section-projets

Ns section projets

imynel committed:
Merge branch 'develop' of github.com:kossakovskaia/everland into NS_section-about

imynel committed:
add adaptive

Commit:
add footer

Commit:
add advantages

Commit:
fix footer section

Commit:
add footer

sashaslab committed:
Merge pull request #44 from kossakovskaia/as_footer

add footer

Sirius-CMa committed:
Merge branch 'develop' of https://github.com/kossakovskaia/everland into develop
111

Sirius-CMa committed:
**Доработки:**
***HTML : ***
*изменена структура БЭМ*
***JS : ***
*переработаны все функции*

Sirius-CMa committed:
Merge pull request #45 from kossakovskaia/an_edit_header_and_menu

An edit header and menu

Sirius-CMa committed:
**Добавдено:**
*плавный скрол*

Sirius-CMa committed:
Merge branch 'an_slowscrolling' into develop

Commit:
fix accordion

Commit:
fix accordion

Commit:
fix conflict accordion

Commit:
fix conflict accordion2

sashaslab committed:
Merge pull request #47 from kossakovskaia/as_advantages

As advantages

Commit:
fix btn-link

sashaslab committed:
Merge pull request #48 from kossakovskaia/btn-link

fix btn-link

imynel committed:
finish layout section-about and merge in develop

imynel committed:
merge section-about

Commit:
fix button

kossakovskaia committed:
Fix layout

kossakovskaia committed:
form wip

kossakovskaia committed:
Merge develop

kossakovskaia committed:
Merge pull request #43 from kossakovskaia/ik_form-section

form wip

kossakovskaia committed:
Fix layout

Commit:
Merge branch 'develop' of github.com:kossakovskaia/everland into as_button

imynel committed:
add section news

imynel committed:
Merge pull request #50 from kossakovskaia/NS_section-news

add section news

Commit:
fix button

kossakovskaia committed:
Fix desktop layout for form section

kossakovskaia committed:
Wip

kossakovskaia committed:
Wip

kossakovskaia committed:
Wip

kossakovskaia committed:
Причесала css

kossakovskaia committed:
Add margins

kossakovskaia committed:
Fux layout

kossakovskaia committed:
Merge pull request #51 from kossakovskaia/ik.form-adaptive

Form layout

kossakovskaia committed:
Fix readme

kossakovskaia committed:
Fix readme

kossakovskaia committed:
Fix readme

kossakovskaia committed:
Fix social icons

kossakovskaia committed:
Fix link

kossakovskaia committed:
Fix margin for devices

kossakovskaia committed:
Add space

kossakovskaia committed:
fix

kossakovskaia committed:
fix

kossakovskaia committed:
fix

Commit:
add button_question

Commit:
fix button

sashaslab committed:
Merge pull request #52 from kossakovskaia/as_button

As button

kossakovskaia committed:
Fix stories btn positioning

kossakovskaia committed:
Fix btns

kossakovskaia committed:
Fix layout

kossakovskaia committed:
Fix layout

kossakovskaia committed:
Fix layout

kossakovskaia committed:
Fix styles

kossakovskaia committed:
Fix styles

kossakovskaia committed:
Fix styles

kossakovskaia committed:
Fix

Savallera reviewed (commented):
Здравствуйте!
Очень хорошая и крепкая работа.
Что можно еще добавить работу чтобы она выгоднее смотрелась в портфолио.
У Вас для этого будет 2 итерации.
Работа будет принята, даже если после второй итерации останутся ошибки.
Не спешите, внимательно ознакомьтесь с чек-листом, обязательно проверяйте проект перед отправкой.
Вы справитесь, желаю вам успехов.

## Критерии

1. Соответствие макету:
    - [x] <!--12.0--> 1.1 Все элементы из макета сверстаны
    - [ ] <!--5.0--> 1.2 Шрифты, размер, цвет и начертания текста соответствуют макету

* https://skr.sh/sI20Gfs778H https://skr.sh/sI241ygy4lB https://skr.sh/sI2ze5oVvo5 https://skr.sh/sI2nZTTEx32

    - [ ] <!--5.0--> 1.3 Ширина элементов и отступы отличаются от макета не более, чем на 30px на десктопной версии и на 10px на остальных размерах экрана, для которых подготовлен макет. Отличия в размерах не должны визуально противоречить макету и выбиваться из заложенной в макет сетки, в этом случае требуется более строгое соответствие размерам макета

* 1440 - https://skr.sh/sI2p3wJRAEb https://skr.sh/sI20rpBtym7 https://skr.sh/sI2msNPI2Vx https://skr.sh/sI2v2VVx5lJ https://skr.sh/sI2oRDDmWKh https://skr.sh/sI2jCfteWHZ https://skr.sh/sI2coVRh2XJ https://skr.sh/sI2VziEpu9p https://skr.sh/sI2KsaEfui6 https://skr.sh/sI2sgEwaBGd https://skr.sh/sI20IfvFT75

* 768 - https://skr.sh/sI2RrgbIPbv https://skr.sh/sI229LWTNkE https://skr.sh/sI2mInVNJxj https://skr.sh/sI2DwFk2la6 https://skr.sh/sI2wjIYphxu https://skr.sh/sI2AvnXHvGP https://skr.sh/sI22sX6s5zy

* 320 - https://skr.sh/sI28ykIcxw5 https://skr.sh/sI2e9CHmb8J https://skr.sh/sI2Mgo77wfo https://skr.sh/sI29O0ijt57 https://skr.sh/sI2o2a2OHq0 https://skr.sh/sI2gRJvwFDa https://skr.sh/sI2NiahkR78 https://skr.sh/sI2bso08utN

    - [ ] <!--3.0--> 1.4 Все блоки соответствуют макету в последних версиях Firefox, Google Chrome или Yandex Browser

* https://skr.sh/sI2oJzYNPbe раскрытые аккордеоны отличаются от макета
* Меню при скролле не закрепляется как предложено в макете https://skr.sh/sI25ftewUJU
* Реализация выпадающего меню по брифу при наведении (Первый уровень доступен при наведении на значок “меню”.), Можно реализовать через стилизацию, пример, https://www.internet-technologies.ru/articles/sozdaem-vypadayuschee-menyu-na-chistom-css.html
* https://skr.sh/sI2bdLQt1FQ на больших экранах проект следует стилизовать под макет 1440, чтобы больше не разъезжался.


2. Адаптив:
    - [ ] <!--4.0--> 2.1 На всех размерах экрана 320px и более отсутствует горизонтальный скролл

* * примерно 320-650, 760-1200 образуется гор.прокрутка при сужении, следует избегать фиксированных размеров, пересмотреть точки брейкпоинта, возможно делать перестройку по раньше https://skr.sh/sI2X7UFBzM8 https://skr.sh/sI2XH4tjOZn (в обощих случаях шалят два блока), а вот с 320-400 уже больше блоков выпадает https://skr.sh/sI2HlObnt1R, просмотрите отступы и размеры.

    - [x] <!--1.0--> 2.2 Установленная максимальная ширина контента соответствует макету
    - [ ] <!--4.0--> 2.3 Вёрстка не ломается между брейкпоинтами

* https://skr.sh/sI25OpKeUhB при сужении блоки наезжают друг на друга

    - [ ] <!--1.0--> 2.4 Брейкпоинты сгруппированы. Если два брейкпоинта имеют небольшую пиксельную разницу в медиа-правиле, они объединены в один

* https://skr.sh/sI22R66B0iz и т.д. брейкпоинты не сгруппированы

    - [x] <!--4.0--> 2.5 Текст не выпадает из блоков на всех размерах экрана 320px и более
    - [x] <!--3.0--> 2.6 Изображения не искажаются и выглядят как в макете на всех размерах экрана
    - [ ] <!--1.0--> 2.7 Одинаковые свойства в разных медиа-правилах не дублируются

* много дублирований (особенно там где несгруппированные медиа)

3. Верстка:
    - [ ] <!--4.0--> 3.1 Нет ошибок валидации [https://validator.w3.org/nu/](https://validator.w3.org/nu/) (Erorrs)

* 19 ошибок валидации

    - [ ] <!--3.0--> 3.2 Используются `<main>`, `<section>`, `<header>`, `<footer>`, `<ul>`, `<nav>`, `<h1-h6>` и `<p>` там, где это необходимо

* https://skr.sh/sI2EIoZEMxn первый заголовок только один (в остальных слайдах можно скорректировать)

    - [x] <!--2.0--> 3.3 Не используются элементы `<b>`, `<br>`, `<i>`
    - [ ] <!--1.0--> 3.4 Отсутствуют лишние `div` обёртки

* donate__container,work-directions__card-desc - выглядят как лишние обертки

    - [x] <!--3.0--> 3.5 Контентные картинки вставлены через `<img>` и в них присутствует `alt` с описанием, соответствующим языку страницы
    - [x] <!--2.0--> 3.6 На странице есть заголовок первого уровня
    - [x] <!--2.0--> 3.7 Заголовки соответствуют иерархии страницы
    - [x] <!--1.0--> 3.8 Корректно задан `viewport`, прописаны `title` и `lang`.
4.  CSS:
    - [x] <!--2.0--> 4.1 Не используется `!important`
    - [x] <!--2.0--> 4.2 Абсолютное позиционирование не используется без непосредственной необходимости
    - [x] <!--4.0--> 4.3 Использование `height` и `max-height` не приводит к выпадению текста или блоков при переполнении
    - [ ] <!--2.0--> 4.4 Любой интерактивный элемент имеет состояние наведения, состояние наведения берется из макета или используется прозрачность

* В выпадающем(и закрепленном) меню ссылки (как и сама кнопка меню) не имеют состояний как в макете https://skr.sh/sI2xcJyHjaA.
* https://skr.sh/sI2xbQoMuAY также кнопки не имеют состояний по макету
* https://skr.sh/sI2398xHuF6 https://skr.sh/sI2utGcwBKw кнопки одинаковые но состояния стилизованы по разному
* ссылки в подвале также не имеют состояний по макету

    - [x] <!--1.0--> 4.5 Для каждого шрифта указаны альтернативные варианты из системных шрифтов и тип шрифта
5. Применение БЭМ:
    - [ ] <!--4.0--> 5.1 Нет нарушения методологии БЭМ: нет элементов без родительского блока, нет элементов элементов, модификаторы не используются без указания того, что они модифицируют

* Есть ошибки по БЭМ. Нужно правильно назвать все классы по методологии БЭМ. Проверить правильность наименования классов можно также тут: https://nglazov.github.io/bem-validator-page/

    - [x] <!--2.0--> 5.2 Не используются селекторы по тегу
    - [x] <!--3.0--> 5.3 Для одинаковых блоков в макете переиспользуется стили одного БЭМ-блока
    - [x] <!--1.0--> 5.4 Числа в названиях классов не используются, за исключением классов для элементов, указанных в руководстве по стилю (Style Guide)
    - [ ] <!--1.0--> 5.5 Модификатор не отражает позицию элемента в разметке (`paragraph_first`)

* footer__column_first footer__column_second

    - [x] <!--1.0--> 5.6 Модификаторы не содержат дубли стилей модифицируемого элемента или блока, только изменяющиеся свойства
    - [ ] <!--2.0--> 5.7 Файловая структура построена по БЭМ nested

* https://skr.sh/sI2NpJBeFV8 https://skr.sh/sI2uZ2h1iRt https://skr.sh/sI2SflDD6T7 и т.д. есть небольшие нарушения структуры

- [x] <!--10.0--> 6. Функциональность на JS работает корректно (слайдеры, бургерное меню или другие требования согласно заданию на проект)
7. Структура проекта и оформление кода
    - [ ] <!--2.0--> 7.1 Соблюдена единообразная файловая структура проекта, например, изображения в папке `images`, шрифты в папке `fonts`, стили в папке `blocks`, скрипты в папке `scripts`, а не разбросаны в разных папках и не хранятся в корне проекта

* Нет папки для файлов скриптов.

    - [x] <!--1.0--> 7.2 Код в проекте отформатирован единообразно
    - [x] <!--1.0--> 7.3 Нет нарушений иерархии отступов в разметке, стилях и файлах скриптов

## Доп. комментарии:

* Выпадающее меню не сдвигает основной контент, а выезжает поверх как выпадающее (на абсолюте).
* в папке шрифтов хранятся лишние файлы, которые в проекте неиспользуются
* button__secondary button__primary - для реализации вариантов кнопок больше подойдут сложные модификаторы, например, блок button имеет темы _theme_primary, _theme_accent, также блок имеет размеры _size_normall, _size_large и т.д., так Вы объедините логически большие блоки в один и применить их можно будет проще
* Помощь агентству и поддержите нас - группу кнопок можно реализовать через группу радио кнопок input type radio, для того, чтобы между ними образовалась связь.
* не у всех кнопок указан тип button type
* Вставка инлайн свг картинок усложняет восприятие кода и ухудшает его читаемость, можно воспользоваться иконкой на псевдоклассе after/before
* work-directions__card_orange, ..., - для цветов нужны сложные модификаторы по типу _ключ_значение
* story-quoted__quote-icon для облегчения разметки можно расположить на псевдоэлементе before
* story-quoted__quote можно использовать тег цитаты blockquote
* story-non-quoted__arrow стрелочки лучше как декоративный элемент, чтобы не нес семантическую нагрузку через тег div (лучше на before/after)
* agency__subheading подзаголовок можно параграфом
* agency__card-text можно сем.заголовк у карточки
* support__form у формы нет имени name
* footer__logo лого свг картинкой лучше загрузить
* footer__menu можно навигационным списком nav
* footer__contact-tel - для телефонов можно оформить ссылками с href только используйте префикс tel:

## Количество баллов: 52

##

Sirius-CMa committed:
***Исправления =>***

*зафиксирована шапка*
*изменены стили*
*изменены медиазапросы*
*добавлены отступы*

*modified:   blocks/header/__logo/header__logo.css*
*modified:   blocks/header/__menu/header__menu.css*
*modified:   blocks/header/header.css*
*modified:   blocks/menu/_open/menu_open.css*
*modified:   blocks/menu/menu.css*

Sirius-CMa committed:
Merge pull request #54 from kossakovskaia/an_menu_edit

***Исправления =>***

Sirius-CMa committed:
**Изменён медиазапрос**
*modified:   blocks/header/__menu-text/header__menu-text.css*

Sirius-CMa committed:
Merge pull request #55 from kossakovskaia/an_menu_text_edit

Изменён медиазапрос

Sirius-CMa committed:
***Ссылки :***

*вынесены отдельным элементом*
*создана БЭМ*
*дополнен index.css*
*исправлена БЭМ _cap*

	renamed:    blocks/header/_cap/header__cap.css -> blocks/header/__cap/header__cap.css
	renamed:    blocks/accordion/__link/accordion_link.css -> blocks/link/_accordion/link_accordion.css
	renamed:    blocks/menu/__link-additional/menu__link-additional.css -> blocks/link/_additional/link_additional.css
	new file:   blocks/link/_disabled/link_disabled.css
	renamed:    blocks/header/__link/header__link.css -> blocks/link/_header/link_header.css
	new file:   blocks/link/_support/link_support.css
	modified:   blocks/link/link.css
	modified:   index.html
	modified:   pages/index.css

Sirius-CMa committed:
Merge pull request #56 from kossakovskaia/an_edit_links

Ссылки :

Sirius-CMa committed:
***Ссылки footer***

*БЭМ*
*стили*
*модификатор*

	deleted:    blocks/footer/__menu-link/footer__menu-link.css
	modified:   blocks/footer/__menu-list/footer__menu-list.css
	new file:   blocks/link/_footer/link_footer.css
	modified:   blocks/link/_support/link_support.css
	modified:   index.html
	modified:   pages/index.css

Sirius-CMa committed:
Merge pull request #57 from kossakovskaia/an_edit_link_footer

Ссылки footer

Commit:
as fix

sashaslab committed:
Merge pull request #58 from kossakovskaia/as_fix

as fix

Sirius-CMa committed:
*промежуточный commit*

Sirius-CMa committed:
*Переделана фиксация header и menu*

	modified:   blocks/menu/menu.css

Sirius-CMa committed:
Merge pull request #59 from kossakovskaia/an_menu_edit_2

An menu edit 2

kossakovskaia committed:
Fix adaptive

Commit:
fix footer support logo

kossakovskaia committed:
Fix adaptive

kossakovskaia committed:
Merge develop

kossakovskaia committed:
Fix links footer logo

kossakovskaia committed:
Fix form

kossakovskaia committed:
Fix form

kossakovskaia committed:
Fix form

kossakovskaia committed:
Merge pull request #60 from kossakovskaia/ik.fixes

Ik.fixes

kossakovskaia committed:
Fix fonts

kossakovskaia committed:
Fix h1

kossakovskaia committed:
Fixes

kossakovskaia committed:
Fixes

Savallera reviewed (approved):
Здравствуйте!
Произведена хорошая работа по предыдущему чек-листу,
уверена, что она будет выгодно смотреться в Вашем портфолио.
Поздравляю Ваша работа принята! Желаю Вам успехов во всех начинаниях!

## Критерии

1. Соответствие макету:
    - [x] <!--12.0--> 1.1 Все элементы из макета сверстаны
    - [ ] <!--5.0--> 1.2 Шрифты, размер, цвет и начертания текста соответствуют макету / 1440 https://skr.sh/sI5AxsOtZKI / 768 https://skr.sh/sI5dToL3Zly https://skr.sh/sI5h7wWfHEn / 320 https://skr.sh/sI550B65VxN https://skr.sh/sI51jXplr1f
    - [ ] <!--5.0--> 1.3 Ширина элементов и отступы отличаются от макета не более, чем на 30px на десктопной версии и на 10px на остальных размерах экрана, для которых подготовлен макет. Отличия в размерах не должны визуально противоречить макету и выбиваться из заложенной в макет сетки, в этом случае требуется более строгое соответствие размерам макета / 1440 https://skr.sh/sI5ZIwwTwz8 https://skr.sh/sI5Fhsq5L56 https://skr.sh/sI5fmvBPM3a https://skr.sh/sI5f0YOxaZ2 / 768 https://skr.sh/sI51PsldVxW https://skr.sh/sI51Gc6CJUy https://skr.sh/sI5uzZP1056 https://skr.sh/sI58we0yaBA / 320 https://skr.sh/sI5WycYE7aI https://skr.sh/sI530lvS1WA https://skr.sh/sI5I01o26Vc https://skr.sh/sI58qaCC9BS https://skr.sh/sI5n1HqYBLn
    - [x] <!--3.0--> 1.4 Все блоки соответствуют макету в последних версиях Firefox, Google Chrome или Yandex Browser
2. Адаптив:
    - [ ] <!--4.0--> 2.1 На всех размерах экрана 320px и более отсутствует горизонтальный скролл / гор скролл до 380 приблизительно сохраняется https://skr.sh/sI50QYgZggt
    - [x] <!--1.0--> 2.2 Установленная максимальная ширина контента соответствует макету
    - [x] <!--4.0--> 2.3 Вёрстка не ломается между брейкпоинтами
    - [ ] <!--1.0--> 2.4 Брейкпоинты сгруппированы. Если два брейкпоинта имеют небольшую пиксельную разницу в медиа-правиле, они объединены в один / https://skr.sh/sI5g1YJIj33 сохраняются не сгруппированные брейкпоинты с разницей менее 150-200пх
    - [x] <!--4.0--> 2.5 Текст не выпадает из блоков на всех размерах экрана 320px и более
    - [x] <!--3.0--> 2.6 Изображения не искажаются и выглядят как в макете на всех размерах экрана
    - [ ] <!--1.0--> 2.7 Одинаковые свойства в разных медиа-правилах не дублируются / сохраняются дублирования в медиа https://skr.sh/sI5kGMVg9h0
3. Верстка:
    - [x] <!--4.0--> 3.1 Нет ошибок валидации [https://validator.w3.org/nu/](https://validator.w3.org/nu/) (Erorrs)
    - [x] <!--3.0--> 3.2 Используются `<main>`, `<section>`, `<header>`, `<footer>`, `<ul>`, `<nav>`, `<h1-h6>` и `<p>` там, где это необходимо
    - [x] <!--2.0--> 3.3 Не используются элементы `<b>`, `<br>`, `<i>`
    - [x] <!--1.0--> 3.4 Отсутствуют лишние `div` обёртки
    - [x] <!--3.0--> 3.5 Контентные картинки вставлены через `<img>` и в них присутствует `alt` с описанием, соответствующим языку страницы
    - [x] <!--2.0--> 3.6 На странице есть заголовок первого уровня
    - [x] <!--2.0--> 3.7 Заголовки соответствуют иерархии страницы
    - [x] <!--1.0--> 3.8 Корректно задан `viewport`, прописаны `title` и `lang`.
4.  CSS:
    - [x] <!--2.0--> 4.1 Не используется `!important`
    - [x] <!--2.0--> 4.2 Абсолютное позиционирование не используется без непосредственной необходимости
    - [x] <!--4.0--> 4.3 Использование `height` и `max-height` не приводит к выпадению текста или блоков при переполнении
    - [x] <!--2.0--> 4.4 Любой интерактивный элемент имеет состояние наведения, состояние наведения берется из макета или используется прозрачность
    - [x] <!--1.0--> 4.5 Для каждого шрифта указаны альтернативные варианты из системных шрифтов и тип шрифта
5. Применение БЭМ:
    - [ ] <!--4.0--> 5.1 Нет нарушения методологии БЭМ: нет элементов без родительского блока, нет элементов элементов, модификаторы не используются без указания того, что они модифицируют / сохраняются ошибки по БЭМ https://nglazov.github.io/bem-validator-page/
    - [x] <!--2.0--> 5.2 Не используются селекторы по тегу
    - [x] <!--3.0--> 5.3 Для одинаковых блоков в макете переиспользуется стили одного БЭМ-блока
    - [x] <!--1.0--> 5.4 Числа в названиях классов не используются, за исключением классов для элементов, указанных в руководстве по стилю (Style Guide)
    - [x] <!--1.0--> 5.5 Модификатор не отражает позицию элемента в разметке (`paragraph_first`)
    - [x] <!--1.0--> 5.6 Модификаторы не содержат дубли стилей модифицируемого элемента или блока, только изменяющиеся свойства
    - [ ] <!--2.0--> 5.7 Файловая структура построена по БЭМ nested / https://skr.sh/sI5hsDkmXwc файлы сложных модификаторов складывают в одну папку по ключу.
- [x] <!--10.0--> 6. Функциональность на JS работает корректно (слайдеры, бургерное меню или другие требования согласно заданию на проект)
7. Структура проекта и оформление кода
    - [x] <!--2.0--> 7.1 Соблюдена единообразная файловая структура проекта, например, изображения в папке `images`, шрифты в папке `fonts`, стили в папке `blocks`, скрипты в папке `scripts`, а не разбросаны в разных папках и не хранятся в корне проекта
    - [x] <!--1.0--> 7.2 Код в проекте отформатирован единообразно
    - [x] <!--1.0--> 7.3 Нет нарушений иерархии отступов в разметке, стилях и файлах скриптов

## Доп. комментарии:

* https://skr.sh/sI5S5KIRNSx при ховере также меняется цвет текста
* https://skr.sh/sI5pwFMubZY https://skr.sh/sI5t7Ke3WTk предпочтительнее мин высота
* https://skr.sh/sI5u41ZtHlj для такого рода модификаторов оправдано создавать сложные по типу ключ значение, он и по структуре лягут в оду папку по ключу и по логике названия будет понятнее.

## Количество баллов: 78

##
